frontend/src/app/main/ui/components/numeric_input.cljs

```diff
@@ -0,0 +1,45 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
+
+(ns app.main.ui.components.numeric-input
+  (:require
+   [rumext.alpha :as mf]
+   [app.main.ui.keyboard :as kbd]
+   [app.common.data :as d]
+   [app.util.dom :as dom]
+   [app.util.object :as obj]))
+
+(mf/defc numeric-input
+  {::mf/wrap-props false}
+  [props]
+  (let [on-key-down
+        (fn [event]
```

Comment:
I think the `mf/use-callback` is missing here.

Reply:
done